keyboards/kingly_keys/romac_plus/readme.md

```diff
@@ -0,0 +1,16 @@
+
+# RoMac+
+
+![RoMac+](https://i.imgur.com/YJ2nCrS.png)
+
+An Upgraded “Plaid” Inspired 12-Key (3x4) Macropad With Fancy Features.
+
+- Keyboard Maintainer: [Garret G.](https://github.com/TheRoyalSweatshirt)
+- Hardware Supported: RoMac+ Rev3.0, Pro Micro, Elite-C, Proton C, BlueMicro.
+- Hardware Availability: Through Online Store [Kingly-Keys](https://kingly-keys.xyz/)  (Stock Varies).
+
+Make example for this keyboard (after setting up your build environment):
+
+    make romac_plus:default
```

Comment:
May be worth using: 

```suggestion
    make romac_plus:default:flash
```

This should look at the configured bootloader and choose the appropriate flashing type.